src/fireedge/src/modules/components/Tables/Enhanced/Utils/GlobalSort.js

```diff
@@ -0,0 +1,108 @@
+/* ------------------------------------------------------------------------- *
+ * Copyright 2002-2025, OpenNebula Project, OpenNebula Systems               *
+ *                                                                           *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may   *
+ * not use this file except in compliance with the License. You may obtain   *
+ * a copy of the License at                                                  *
+ *                                                                           *
+ * http://www.apache.org/licenses/LICENSE-2.0                                *
+ *                                                                           *
+ * Unless required by applicable law or agreed to in writing, software       *
+ * distributed under the License is distributed on an "AS IS" BASIS,         *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  *
+ * See the License for the specific language governing permissions and       *
+ * limitations under the License.                                            *
+ * ------------------------------------------------------------------------- */
+import PropTypes from 'prop-types'
+import { memo, ReactElement, useEffect, useMemo } from 'react'
+
+import { MenuItem, MenuList, Stack } from '@mui/material'
+import { ArrowDown, ArrowUp, SortDown } from 'iconoir-react'
+import { UseSortByInstanceProps, UseSortByState } from 'opennebula-react-table'
+
+import { T, STYLE_BUTTONS } from '@ConstantsModule'
+import HeaderPopover from '@modules/components/Header/Popover'
+import { Translate } from '@modules/components/HOC'
+
+/**
+ * Render all selected sorters.
+ *
+ * @returns {ReactElement} Component JSX
+ */
+const GlobalSort = memo(
+  (useTableProps) => {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This route is missing authorization checks. Without proper authorization, sensitive routes may be exposed to unauthorized users, leading to potential data breaches or unauthorized actions. Ensure that authorization middleware or handlers are applied. For more information on missing authorization vulnerabilities, refer to [CWE-862](https://cwe.mitre.org/data/definitions/862.html).